apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -1,13 +1,52 @@
 import { findAllNewAppeals } from './validation.service.js';
 
+/**
+ * Given an item object transform it into - govuk/components/table/macro.njk" import govukTable format
+ *
+ * @param {*} item to do
+ * @returns {any} to do
+ */
+function makeGovTableRow(item){
```

Comment:
I think this function can be part of the actual controller function. Is not used anywhere else. I can even argue that this could be refactored to be the function of a reducer traversing the appeals data, but even so should be part of the controller.

Reply:
Yes it originally was a foreach reducer , but lint complained and QuickFix on IDE made it into a function. Will move back and put a disable lint comment against it

Reply:
Not for each, I meant `array.reduce()` but its OK it was just a thought about it, you can keep the for loop.